core/src/services/fs/lister.rs

```diff
@@ -93,17 +116,38 @@ impl oio::BlockingList for FsLister<std::fs::ReadDir> {
         // (no extra system calls needed), but some Unix platforms may
         // require the equivalent call to symlink_metadata to learn about
         // the target file type.
-        let file_type = de.file_type().map_err(new_std_io_error)?;
+        let default_meta = self.op.metakey() == Metakey::Mode;
```

Comment:
Hi, please remove the upper comment before `de.file_type()`.

Reply:
done~